docs/CONFIG-VARS.md

```diff
@@ -45,6 +37,16 @@ Supported configuration variables are listed in the table below.  All variables
 | V4_CFG_SITEDEFAULT | Path to sitedefault file | string | | false | When not set [sitedefault](examples/sitedefault.yaml) is used | viya |
 | V4_CFG_SSSD | Path to sssd file | string | | false | | viya |
 
+## Cloud info
+
+| Name | Description | Type | Default | Required | Notes | Actions |
+| :--- | ---: | ---: | ---: | ---: | ---: | ---: |
+| PROVIDER | Cloud provider | string | | true | [aws,azure,gcp,custom] | baseline, viya |
+| CLUSTER_NAME | Name of the k8s cluster | string | | true | | baseline, viya |
+| NAMESPACE | K8s namespace in which to deploy | string | | true | | baseline, viya, viya-monitoring |
+| V4_CFG_CLOUD_SERVICE_ACCOUNT_NAME | Cloud service account | string | | false | See [ansible cloud authentication](user/AnsibleCloudAuthentication.md) | viya |
```

Comment:
I believe these items are provider specific and need to live in their own area. We had this prior and now it's been moved here. They are out of place.

Reply:
But the original note for these new values said for now the only valid scenario is for GCP. Implying they could be used for future cloud providers correct? Will these variables will ONLY ever be used for GCP?

Reply:
They could and are expected to be used for other clouds

Reply:
Given what Norm described I vote for keeping them in cloud info.